runtime/doc/lsp.txt

```diff
@@ -25,24 +25,37 @@ Nvim provides an LSP client, but the servers are provided by third parties.
 Follow these steps to get LSP features:
 
 1. Install language servers using your package manager or by following the
-   upstream installation instruction. You can find language servers here:
+   upstream installation instructions. You can find language servers here:
    https://microsoft.github.io/language-server-protocol/implementors/servers/
 
 2. Configure the LSP client per language server. See |vim.lsp.start()| or use
    this minimal example as a guide: >lua
 
-    vim.lsp.start({
-      name = 'my-server-name',
-      cmd = {'name-of-language-server-executable'},
-      root_dir = vim.fs.root(0, {'setup.py', 'pyproject.toml'}),
+    -- Create an event handler for the FileType autocommand
+    vim.api.nvim_create_autocmd('FileType', {
+      -- This handler will fire when the buffer's 'filetype' is "python"
+      pattern = 'python',
+      callback = function(ev)
+        -- The buffer number this event is associated with
```

Comment:
I would recommend structuring these comments around the _intention_ rather than a technical description. In this case, I would actually _not_ define this here but use the event structure where it is needed (see below).

Reply:
The intent here was to (hopefully) avoid confusion between "current buffer" and "buffer associated with the autocommand". Pulling the `bufnr` declaration out allows us to add a separate comment explaining this distinction. But perhaps this is not an important distinction for new users.